.deployment

```diff
@@ -0,0 +1,2 @@
+[config]
+command = deploy.cmd
```

Comment:
why do we need this?

Reply:
That is the custom build script that disables func pack when the collector installs and updates. It allows us to use the new sdks without worry of funcpack breaking them.